tests/linearGaussian_snpe_test.py

```diff
@@ -41,12 +41,12 @@
 )
 
 
-@pytest.mark.parametrize("npe_method", [NPE_A, NPE_C])
+@pytest.mark.parametrize("npe_method_str", ["npe_a", "npe_b", "npe_c"])
 @pytest.mark.parametrize(
     "num_dim, prior_str",
     ((2, "gaussian"), (2, "uniform"), (1, "gaussian")),
 )
-def test_c2st_npe_on_linearGaussian(npe_method, num_dim: int, prior_str: str):
+def test_c2st_npe_on_linearGaussian(npe_method_str, num_dim: int, prior_str: str):
```

Comment:
Here we only test single-round, where `NPE_B` and `NPE_C` are the same - I don't think we need to add `npe_b` to the test here. I would keep `NPE_A` as it explicitly tests MDNs.